2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2021 @@
+environment. The long-term solution in order to gain sufficient software
+development **skills** should be education that is included early in the
+career path, ideally already at the Bachelor level. For the time being
+however, efforts involving workshops and seminars that provide easy
+access to hands-on training on software-related questions should be
+promoted and supported as much as possible.
+
+It is important to provide an environment where **communities** can form
+and flourish by allocating resources for research software development
+and for building communities around it (Katz et al. 2019; Iaffaldano et
+al. 2019; Jiménez et al. 2017). The identification with a community of
+like-minded people and personal action (Allen et al. 2017) can lead to a
+permanent establishment of sustainable research software as a valuable
+research output. Thus, research institutions as well as funding agencies
+should not only be open-minded regarding existing volunteer
+organizations, but should actively promote the creation of such groups.
+
+How can Research Software be Sustainably Funded?
+================================================
+
+> *Hal9k* has grown into a widely used software in researchonomics, and
+> Kim is proactively asked to apply for - and is subsequently awarded -
+> a permanent RSE position at the institute for researchonomy at UofA,
+> based on her work on the library. She works closely with the central
+> RSE team, but mostly due to bureaucracy and the high demand for her
+> library, Kim does not have enough time to maintain and further develop
+> *hal9k* alone anymore. Together with the dean she develops a course
+> for the researchonomics curriculum which teaches data processing with
+> *hal9k*. As a lesson from her own career, she starts the course with
+> sessions on the Hash shell, version control with Tig, Boa, and two
+> whole sessions on basics of sustainable software development. This is
+> very fruitful, and due to the implementation of a new research
+> software funding scheme at UofA, Kim is able to hire one of the course
+> students, who has shown great research software engineering skills
+> straight into a long-term position at her institute, where they focus
+> on the maintenance and development of *hal9k*, work with the computing
+> center to support *hal9k*-based supercomputing on a new, dedicated
+> FGPA cluster, develop training materials for external users, and
+> organize the yearly *hal9k* users and developers conference. Kim gets
+> to travel the world to visit researchonomics groups who are using
+> *hal9k*.
+
+Requirements
+------------
+
+Sustainable funding for research software boils down to funding the four
+main pillars enabling sustainable software development: i) Personnel
+with expertise in research software development; ii) Infrastructure for
+developing, testing, validating, and benchmarking research software, and
+distributed versioning systems for collaborative software development;
```

Comment:
how about execution environments? Having a jupyter notebook server at hand is useful too. As mention previously, telemetry infrastructure could be needed as well

Reply:
certainly useful elements, but I would like to avoid adding more and more details here and deluting the main message of funding personal, infrastructure and training